docs/topics/what-is-the-toolkit.adoc

```diff
@@ -11,14 +13,23 @@
 [discrete]
 == What is the {ProductName}?
 
-The {ProductName} ({ProductShortName}) is an extensible and customizable rule-based tool that simplifies the migration of Java applications.
+The {ProductName} ({ProductShortName}) is an extensible and customizable rule-based tool that simplifies the migration and modernization of Java applications.
 
-The {ProductShortName} examines application artifacts, including project source directories and application archives, then produces an HTML report that highlights areas needing changes. {ProductShortName} can migrate Java applications from earlier versions of Red Hat JBoss Enterprise Application Platform or from other application servers, such as Oracle WebLogic Server or IBM WebSphere Application Server.
+{ProductShortName} examines application artifacts, including project source directories and application archives, and then produces an HTML report highlighting areas needing changes. {ProductShortName} supports many migration paths including the following examples:
+
+* Migrating from enterprise application servers to Red Hat JBoss Enterprise Application Platform
+* Containerizing applications and making them cloud-ready
+* Migrating from Spring Boot to Quarkus
+* Updating OpenJDK version

```

Comment:
Change to either "Updating an OpenJDK version" or "Updating OpenJDK versions". The singular without an article looks odd.